io.openems.edge.levl.controller/src/io/openems/edge/levl/controller/LevlControlRequest.java

```diff
@@ -0,0 +1,140 @@
+package io.openems.edge.levl.controller;
+
+import com.google.gson.JsonObject;
```

Comment:
Now I use JsonUtils within the parseFields method. However, the OpenEMS'es own JsonrpcRequest.getParams() returns an gson JsonObject.
If there's a OpenEMS way to avoid this, I would appreciate a short hint :)